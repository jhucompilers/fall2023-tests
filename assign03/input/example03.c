void incr(int *p);

int main(void) {
  int a;
  a = 1;
  incr(&a);
  return a;
}
