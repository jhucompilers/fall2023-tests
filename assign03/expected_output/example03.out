0|incr|function|function (pointer to int) returning void
0|main|function|function () returning int
2|a|variable|int
